### PR TITLE
feat: create classroom list component

### PR DESCRIPTION
**Pull request by carissa-tang**
## Notion ticket link

[Classroom list component](https://www.notion.so/uwblueprintexecs/Classroom-list-component-8951d0e290674fd4a1c3a0e0ceca4537?pvs=4)

## Implementation description
* Create the frontend component used to display classroom information

## Steps to test
1. Use preview url to visit `/component-library`
image

## What should reviewers focus on?
* Match with Figma and any missing details

## Checklist
- [ ] My PR name is descriptive and in imperative tense
- [ ] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [ ] I have run the appropriate linter(s)
- [ ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: frontend/src/components/pages/teacher/classroom/ClassroomCard.tsx [hunk 1]
```diff
@@ -0,0 +1,90 @@
+import React from "react";
+import { Box, HStack, Tag, Text } from "@chakra-ui/react";
+
+import { Grade } from "../../../../APIClients/types/UserClientTypes";
+import {
+  BarChart2Icon,
```
Comment (carissa-tang) [inline frontend/src/components/pages/teacher/classroom/ClassroomCard.tsx]: rename to BarChartIcon

File: frontend/src/components/pages/teacher/classroom/ClassroomCard.tsx [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+          </HStack>
+        );
+      })}
+      {!!activeAssessments && (
+        <Tag
+          bg="grey.100"
+          borderRadius="1000px"
+          color="grey.300"
+          mt="0.65em"
+          p="0.5em 1em"
```
Comment (carissa-tang) [inline frontend/src/components/pages/teacher/classroom/ClassroomCard.tsx]: move textStyle="mobileParagraph" to tag

Comment (carissa-tang) [reply frontend/src/components/pages/teacher/classroom/ClassroomCard.tsx]: refactor w/ questionTag

**Review by joyce-shi — APPROVED** [review]
LGTM - just the **Tag** refactor that we talked about (:

**Pull request merged**